run_unittests.py

```diff
@@ -4729,10 +4770,20 @@ def test_cross_find_program(self):
         testdir = os.path.join(self.unit_test_dir, '11 cross prog')
         crossfile = tempfile.NamedTemporaryFile(mode='w')
         print(os.path.join(testdir, 'some_cross_tool.py'))
-        crossfile.write('''[binaries]
-c = '/usr/bin/cc'
-ar = '/usr/bin/ar'
-strip = '/usr/bin/ar'
+
+        if is_solaris():
+             c = '/usr/bin/gcc'
```

Comment:
__[Flake8]__ [E111] indentation is not a multiple of four ([link](https://sider.review/gh/repos/19784232/pulls/5008#issue-3817077))

<sub>posted by Sider</sub>